### PR TITLE
Impove images and data loading in leaderboad and footer

### PR DESCRIPTION
**Pull request by ukin-level**
- [x] Fix image size to fix ui bug when users visit for the first time in Leaderboard.
- [x] Show token price info when data loading success.

### CODE REVIEW

File: src/views/leaderboard/conponents/TopTrader.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const TopTrader: React.FC<TopTraderProps> = ({ items, loading }) => {
             </p>
           </>
         )}
-        <img src={img2st} className="w-145px xl:w-210px mr--30px xl:mr--50px" />
+        <img src={img2st} className="w-145px xl:w-210px h-66px xl:h-96px mr--30px xl:mr--50px" />
```
Comment (h3xag0nx) [inline src/views/leaderboard/conponents/TopTrader.tsx]: image size should be set as attribute for browser to render before css loaded

**Review by h3xag0nx — CHANGES_REQUESTED** [review]

File: src/views/leaderboard/conponents/TopTrader.tsx (hunk 1 above)
Comment (0xhari-s) [reply src/views/leaderboard/conponents/TopTrader.tsx]: right, @ukin-level plz update

Comment (0xleeaki) [reply src/views/leaderboard/conponents/TopTrader.tsx]: Thank's @20xlevel. I have updated my pull request.

Comment (ukin-level) [reply src/views/leaderboard/conponents/TopTrader.tsx]: Thank's @20xlevel, I have updated my pull request.

Comment (0xhari-s) [reply src/views/leaderboard/conponents/TopTrader.tsx]: it's good to go

**Pull request merged**